proposed/2021/MachineReadableOutput.md

```diff
@@ -0,0 +1,807 @@
+                "resolvedVersion": "1.1.2",
+                "latestVersion": "1.1.4",
+              }
+            ],
+            "transitivePackages": [
+              {
+                "id": "Microsoft.CSharp",
+                "resolvedVersion": "4.0.1",
+                "latestVersion": "4.7.0",
+              }
+      ...
+            ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### `> dotnet list package --include-transitive --deprecated --framework net5.0`
+
+The following sources were used:
+   https://api.nuget.org/v3/index.json
+   https://apidev.nugettest.org/v3-index/index.json
+
+No packages were found for the project `MyProjectA` given the specified frameworks.
+Project `MyProjectB` has the following deprecated packages
+   [net5.0]:
+   Top-level Package      Requested   Resolved   Reason(s)   Alternative
+   > NuGet.Core           2.13.0      2.13.0     Legacy
+
+   Transitive Package          Resolved              Reason(s)   Alternative
+   > NuGet.Packaging.Core      4.8.0-preview3.5278   Legacy      NuGet.Packaging >= 0.0.0
+
+#### `> dotnet list package --include-transitive --deprecated --framework net5.0 --format json`
+
+```json
+{
+  "version": 1,
+  "parameters": "-include-transitive --outdated --framework net5.0",
+   "sources": [
+    "https://api.nuget.org/v3/index.json",
+    "https://apidev.nugettest.org/v3-index/index.json"
+  ],
+  "projects": [
+    {
+      "path": "src/lib/MyProjectA.csproj",
+      "frameworks": [
+      ]
+    },
+    {
+      "path": "src/lib/MyProjectB.csproj",
+      "frameworks": [
+        {
+          "framework": "net5.0",
+          "topLevelPackages": [ 
+              {
+                "id": "NuGet.Core",
+                "requestedVersion": "2.13.0",
+                "resolvedVersion": "2.13.0",
+                "deprecationReasons": ["Legacy"]
+              }
+            ],
+            "transitivePackages": [
+              {
+                "id": "Microsoft.CSharp",
+                "resolvedVersion": "4.0.1",
+                "deprecationReasons": ["Legacy"],
+                "alternativePackage": {
+                  "id": "NuGet.Packaging",
+                  "versionRange": "[0.0.0,)"
+                }
+              }
+      ...
+            ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+## Compatibility
+
+We start with `version 1`, as long as we don't remove or rename then it'll be backward compatible. In case [we change version](https://stackoverflow.com/a/13945074) just add new properties, keep old ones even it's not used.
+By default `--format json` will output latest schema version of json, but we can set older version of output like `--output-version 1`, intention behind is we don't want to customer CI build script because sdk/nuget version is upgraded and keep it predictable.
+
+## Out-of-scope
+
+* We won't support saving the machine-readable output to disk as part of this spec. The work-around is for the consumer to read from the console's stdout stream.
```

Comment:
Consider simplifying these. 

Ex. 

* Saving the output to disk. 
* Any other additions, such as `--parsable`. 
* License information (link issue if created). 

No need to elaborate too much and it's super straightforward to know what's supported.